projects/BFDR.Tests/BirthRecord_Should.cs

```diff
@@ -1432,6 +1479,7 @@ private string FixturePath(string filePath)
     public void SetAttendantAfterParse()
     {
         BirthRecord sample1 = new BirthRecord(File.ReadAllText(TestHelpers.FixturePath("fixtures/json/BasicBirthRecord.json")));
+        //name
```

Comment:
there should be a test that parses a json record with attendant data already defined in the json to make sure the code can parse the field using the extension